app/gui/view/component-browser/component-list-panel/grid/src/lib.rs

```diff
@@ -505,6 +524,7 @@ impl component::Frp<Model> for Frp {
             grid_position <- style_and_content_size.map(Model::grid_position);
             eval grid_position ((pos) model.grid.set_xy(*pos));
             grid_scroll_frp.set_corner_radius_bottom_right <+ all(&corners_radius, &style.init)._0();
+            grid_scroll_frp.set_corner_radius_bottom_left <+ all(&corners_radius, &style.init)._0();
```

Comment:
The expression `all(&corners_radius, &style.init)._0()` is repeated, creating unnecessary FRP nodes. It should be given a name and used for both corners.

```suggestion
            init_corners_radius <- all(&corners_radius, &style.init)._0();
            grid_scroll_frp.set_corner_radius_bottom_right <+ init_corners_radius;
            grid_scroll_frp.set_corner_radius_bottom_left <+ init_corners_radius;
```